docs/triggers/shortcut-triggers.md

```diff
@@ -0,0 +1,122 @@
+## Shortcut Triggers
```

Comment:
I think shortcut makes sense in this case since the developer will use `type: "shortcut"`. Link trigger just so happens to be the only type of shortcut trigger we currently allow.